### PR TITLE
feat: extend punks2023 metadata

### PR DESCRIPTION
**Pull request by haseebahmad109**

### CODE REVIEW

File: packages/indexer/src/metadata/extend/punks2023/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { TokenMetadata } from "@/metadata/types";
+
+const metadataBaseURI = "https://punks2023.com/api/marketplace";
```
Comment (ipeleg) [inline packages/indexer/src/metadata/extend/punks2023/index.ts:4]: @haseebahmad109 let's move this const to be inside the extend function

File: packages/indexer/src/metadata/extend/punks2023/index.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+export const extend = async (metadata: TokenMetadata) => {
+  const response = await axios.get(`${metadataBaseURI}/${metadata.tokenId}`);
+
+  const { attributes: customAttributes } = response.data;
```
Comment (ipeleg) [inline packages/indexer/src/metadata/extend/punks2023/index.ts:9]: What is customAttributes? shouldn't it be defined?

**Review by ipeleg — CHANGES_REQUESTED** [review]

File: packages/indexer/src/metadata/extend/punks2023/index.ts (hunk 1 above)
Comment (haseebahmad109) [reply packages/indexer/src/metadata/extend/punks2023/index.ts]: @ipeleg 
Fixed and moved to extend function.

File: packages/indexer/src/metadata/extend/punks2023/index.ts (hunk 2 above)
Comment (haseebahmad109) [reply packages/indexer/src/metadata/extend/punks2023/index.ts]: @ipeleg 

Type for customAttributes is now defined.

**Review by ipeleg — APPROVED** [review]

**Pull request merged**